docs/topics/modular-testing.md

```diff
@@ -72,8 +72,7 @@ class MultiMapTest {
     @Test
     fun modularTest() = ModelCheckingOptions()
         .addGuarantee(forClasses(ConcurrentHashMap::class).allMethods().treatAsAtomic())
-        // Note that with the atomicity guarantees set, Lincheck can examine all possible interleavings,
-        // so the test successfully passes when the number of invocations is set to `Int.MAX_VALUE`
+        // Note that with the atomicity guarantees set, Lincheck can examine all possible interleavings.
```

Comment:
Or should the example be modified to use a fixed implementation of the `MultiMap::add` method so that the test always succeeds?